jcheck/src/main/java/org/openjdk/skara/jcheck/ReviewersCheck.java

```diff
@@ -67,7 +67,7 @@ private String nextRequiredRole(String role, Map<String, Integer> count) {
 
     @Override
     Iterator<Issue> check(Commit commit, CommitMessage message, JCheckConfiguration conf, Census census) {
-        if (commit.isMerge() || utils.addsHgTag(commit)) {
+        if (utils.addsHgTag(commit)) {
```

Comment:
We can't just remove this default behavior in jcheck itself. Then the cli version of jcheck would always require reviewers for a merge commit. We are going to need a new setting in `.jcheck/conf` for this and the PR bot would override it when creating the jcheck instance.

Reply:
Ok, got it. If we have to make changes to `.jcheck/conf`. I will rework on this issue. It would be much more easy than current solution.

Reply:
I think you misunderstood me. It needs to be possible to configure this from `.jcheck/conf`, but we still want to also be able to control it in the PR bot configuration for a specific repository. In the latter case it would be inserted as a config override when create the jcheck instance.

We want to enable this for the jdk main repository, but it should not get inherited by all downstream and project repos.